DependencyInjection/Compiler/SerializerPass.php

```diff
@@ -24,8 +24,11 @@ class SerializerPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
```

Comment:
Since v1.3 of jms/serializer-bundle the service definition contains 2 arguments, so we have to be aware of it and pass the second arg to the constructor of `JsonApiSerializationVisitor`